docs/config.md

```diff
@@ -0,0 +1,38 @@
+# Adding an  extra category to the search bar
+
+# To start of here is an example config
+
+		"ImqsServerMirror": {
+			"Tables": {
+			"AssetComponent": {
+					"Categories": [
+						"Assets"
+					],
+					"FriendlyName": "Facility Name",
+					"Fields": [
+						{
+							"FriendlyName": "ComponentID",
+							"Field": "ComponentID",
+							"Order": "1"
+						},
+						{
+							"FriendlyName": "Facility Name",
+							"Field": "AssetFacilityName",
+							"Order": "2"
+						}
+					]
+				},
```

Comment:
Add numbers after certain entries to make it easier to reference them in the rest of the file. eg:
```json
{
    "ImqsServerMirror": { // (1)
        "Tables": {
	    "AssetComponent": { // (2)
```
Then you can refer to (1) in the file and people will have a good idea of what you are talking about :).